prow/config.yaml

```diff
@@ -1315,6 +1315,7 @@ presubmits:
     context: pull-kubernetes-e2e-gke-gci
     rerun_command: "/test pull-kubernetes-e2e-gke-gci"
     trigger: "(?m)^/test pull-kubernetes-e2e-gke-gci,?(\\s+|$)"
+    run_if_changed: '^(cluster/gce).*$'
```

Comment:
We've also had breakages come from `cluster/addons`. Would it be too broad to add that?

Reply:
are there people from outside google touch those cluster/addons? I guess very few if any. i'm inclined to add it.

Reply:
Since it would have caught one of the recent self-inflicted breakages, I say we add it for now. We can always readjust the trigger if we see unintended friction.